src/CppAst/CppModelBuilder.cs

```diff
@@ -235,45 +329,44 @@ private CXChildVisitResult VisitMember(CXCursor cursor, CXCursor parent, void* d
                 case CXCursorKind.CXCursor_ClassDecl:
                 case CXCursorKind.CXCursor_StructDecl:
                 case CXCursorKind.CXCursor_UnionDecl:
-                {
-                    bool isAnonymous = cursor.IsAnonymous;
-                    var cppClass = VisitClassDecl(cursor, data);
-                    // Empty struct/class/union declaration are considered as fields
-                    if (isAnonymous)
                     {
-                        Debug.Assert(string.IsNullOrEmpty(cppClass.Name));
-                        var containerContext = GetOrCreateDeclarationContainer(parent, data);
-
-                        // We try to recover the offset from the previous field
-                        // Might not be always correct (with alignment rules),
-                        // but not sure how to recover the offset without recalculating the entire offsets
-                        var offset = 0;
-                        var cppClassContainer = containerContext.Container as CppClass;
-                        if (cppClassContainer is object && cppClassContainer.Fields.Count > 0)
+                        bool isAnonymous = cursor.IsAnonymous;
```

Comment:
I believe this code has been shifted to the right when editing this file. It is difficult to see if there is a change in this. Could you revert the shift?